chain/consensus/mirfbt/utils.go

```diff
@@ -0,0 +1,34 @@
+package mirbft
+
+import (
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/lotus/chain/consensus/common"
+	"github.com/filecoin-project/lotus/chain/types"
+)
+
+func parseTx(tx []byte) (interface{}, error) {
+	ln := len(tx)
+	if ln <= 2 {
```

Comment:
Why <= 2?

Reply:
Actually < 2. Because the last byte is message type. It would be great to improve this validation.